doc/doc-site/docs/overview/orders.md

```diff
@@ -514,98 +431,91 @@ Market orders and swaps allow users to specify precise trade amounts down to a s
 
 :::
 
-Hence the nominal price of 17792.27 `USD` per `BTC` must be truncated to either 17792.26 or 17792.28.
-For the latter case, this entails an integer price of 889614:
+Next, consider a liquid staking derivative `sAPT` trading against `APT`, `sAPT/APT`, both having 8 decimals.
+Here, very-high price granularity is necessary because the pair is "mean reverting" (to a ratio of 1):
+
+| Variable          | Value          |
+| ----------------- | -------------- |
+| Decimal lot size  | 0.01 `sAPT`    |
+| Decimal tick size | 0.000001 `APT` |
+| Base decimals     | 8              |
+| Quote decimals    | 8              |
+
+Let's check the minimum and maximum price here using the SDK:
 
 ```python
->>> price_decimal = dec('17792.28')
->>> price_integer = int(price_decimal / tick_size_decimal)
->>> price_integer
-889614
+>>> lot_size = "0.01"
+>>> tick_size = "0.000001"
+>>> get_min_quote_per_base_nominal(lot_size, tick_size)
+0.0001
+>>> get_max_quote_per_base_nominal(lot_size, tick_size)
+429496.7295
 ```
 
-Next, consider a liquid staking derivative `sAPT` trading against `APT`, `sAPT/APT`, both having 8 decimals.
-Here, high price precision may be appropriate:
-
-| Variable          | Value                     |
-| ----------------- | ------------------------- |
-| Decimal lot size  | 0.01 `sAPT`               |
-| Decimal tick size | 0.000001 `APT` per `sAPT` |
-| Base decimals     | 8                         |
-| Quote decimals    | 8                         |
-| Decimal price     | 1.000012 `APT` per `sAPT` |
+This gives us a price precision of 1/100th of a penny, with room to go even more precise should we desire.
+However we recall that the most granular possible price isn't necessarily desirable due to almost every order then occupying its own price level, cluttering user interfaces.
+Some coarseness is desirable for that reason.
+Although it is possible for user interfaces to summarize orders into price levels that don't actually exist, this would diverge what the user sees from reality in a way that can't be guaranteed safe.
```

Comment:
"this approach would represent to the user information that is not accurate, and is not recommended"